superset-frontend/src/explore/components/controls/SelectControl.test.jsx

```diff
@@ -153,11 +153,12 @@ describe('SelectControl', () => {
     });
 
     describe('when select has a sortComparator prop', () => {
-      it('does not add add order key and sorts by sortComparator', () => {
```

Comment:
This test case expect the child component `Select` to update a prop passed down from `SelectComponent`, which is a bad practice as all React props should be immutable.

I didn't see anything breaking after removing this behavior. @corbinrobb @michael-s-molina do you remember why this was needed?